### DISCUSSION TITLE
correct way to update?

### DISCUSSION THREAD

**Discussion by gnagga** [seerge/g-helper#861] [General]
I'm confused on how the update process works. Is it a fully portable app, so that I have to move the downloaded .exe on top of the previous one in the folder i made for it in C:\programms?
or do i just have to run it once from downloads and it somehow finds and updates the old exe by itself?

**Comment by seerge** [comment]
@gnagga if you update manually, you can just replace exe. For major updates app will offer autoupdate, and replace own exe by itself.
